modules/customize-certificates-api-add-named.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/api-server.adoc
+
+[id="add-named-api-server-{context}"]
+= Add an API server named certificate
+
+The default API server certificate is issued by an internal {product-title} 
+cluster CA. Additional certificates may be added to the API server to send
+based on the client's requested URL.
+
+.Prerequisites
+
+* You must have the certificate and key for the client's URL. 
```

Comment:
Might benefit from a little more detail:

The cert and key files need to be in PEM format.
The cert needs to be issued for the URL that is used by the client to reach the API server. This means it needs a SAN extension (subjectAltName) for the URL.

Reply:
I've made these changes for the `API server named certificate` section. Does the default certificate require the PEM format as well?